### PR TITLE
(Feat): Calculate f1 score

### PR DESCRIPTION
**Pull request by pgiank28**

### CODE REVIEW

File: evaluators/rank/f1ScoreEvaluator/__init__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from jina.executors.evaluators.rank import BaseRankingEvaluator
+
+class f1ScoreEvaluator(BaseRankingEvaluator):
+    """
+    :class:`f1ScoreEvaluator` Gives the f1 score of a result given the groundtruth..
+    """
+
+    def __init__(self, *args, **kwargs):
```
Comment (JoanFM) [inline evaluators/rank/f1ScoreEvaluator/__init__.py:8]: for our tests to pass, we need to provide a valid `eval_at` default, we can pass it in the `config.yml` or directly in the `__init__` of our class which we are doing right now in other PRs.

**Review by JoanFM — COMMENTED** [review]
LGTM! Just minor comment as it will most likely not pass the CI

**Comment by JoanFM** [comment]
Bad commit message, please refer to:
- [Commit Message Guideline for the First Time Contributor](https://github.com/jina-ai/jina/issues/553)
- [Contributing Guideline](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md)
#553 :bookmark: Commit Message Guideline for the First Time Contributor
jina-ai/jina | Jun 21st | Added by GitHub
CONTRIBUTING.md
jina-ai/jina | Added by GitHub

**Comment by JoanFM** [comment]
Thanks for your contribution. Unfortunately, as you didn't set up your local git config correctly, the author of your commit does not match with your Github account. This poses problems:
Your contribution can not be credited, and will not show up in the list of contributors (as bad git config = anonymous)
We can not accept CLA for anonymous contribution.
Please note that this does not only affect your contribution to Jina, but also affects all your contribution to anything on Github. So better fix it, once for all.
Here is what you can do:
```
git config user.name "Correct new name"
git config user.email "correct@new.email" -> match with your Github registered email address
```
Then fix the commit message by
```
git commit --amend --author="Author Name <email@address.com>" --no-edit
git push --force
```

**Comment by pgiank28** [comment]
Thanks for the advice, i am working on proper commit messages and global configuration.

**Pull request closed without merging**